io-engine/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -211,6 +211,12 @@ impl NexusNvmeParams {
     pub fn set_preempt_policy(&mut self, preempt_policy: NexusNvmePreemption) {
         self.preempt_policy = preempt_policy;
     }
+    /// Check if reservations are disabled.
+    pub fn disabled(&self) -> bool {
```

Comment:
Can we use here a name like 'reservations_disabled()' to denote that it deals with reservations ?
Also, as a nit, 'enabled()' looks a bit more natural to me than '.disabled()', but it just a suggestion here.

Reply:
Ah I thought the type name here was NvmeReservations which is why I left out reservations but looks like it's only NvmeParams, so I'll rename to reservations_enabled().